Assets/MixedRealityToolkit/Providers/BaseBoundarySystem.cs

```diff
@@ -22,6 +22,36 @@ public abstract class BaseBoundarySystem : BaseCoreSystem, IMixedRealityBoundary
             ExperienceScale scale) : base(profile)
         {
             Scale = scale;
+            ReadProfile();
```

Comment:
Does this not work if left in `Initialize`? I think I remember we had discussed putting profile reads in `Initialize` so things like `Reset` would re-read a profile if it changed.

Reply:
it does not. we may want to consider having Reset read the profile as well as the ctor.

Reply:
Huh. Would be nice if we could read the profile in one place though

Reply:
We solved this offline. Thanks again for looking at it with me!